bundles/org.openhab.binding.publictransportswitzerland/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="publictransportswitzerland"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="stationboard">
+		<label>Public Transport Switzerland - Stationboard</label>
+		<description>Upcoming departures</description>
+
+		<channels>
+			<channel id="csv" typeId="csv" />
+			<channel id="json" typeId="json" />
```

Comment:
I don't believe an end user needs a channel with the JSON response from the API.
I think you could replace this channel by a DEBUG log that will log the JSON response.

Reply:
Resolved with e463aac3c9